django_stubs_ext/django_stubs_ext/patch.py

```diff
@@ -8,6 +8,7 @@
 from django.contrib.syndication.views import Feed
 from django.core.files.utils import FileProxyMixin
 from django.core.paginator import Paginator
+from django.db.models.expressions import ExpressionWrapper, NegatedExpression
```

Comment:
Oof, this import will fail with Django 4.1 and earlier, since the class only appeared in 4.2.

Also we clearly have a blind spot in our CI since it didn't detect this.

@sobolevn Any ideas how to handle backwards compatibility in django-stubs-ext monkeypatching?

Reply:
PR #1552 will add CI coverage for such surprises.

Reply:
Anyway I'd remove `NegatedExpression` monkeypatching for now, to unblock this PR again.

I've created issue #1555 to solve it in the future.